src/components/Popup.js

```diff
@@ -20,6 +28,7 @@ export default class Popup {
   }
 
   setEventListeners() {
+    this._submitBtn = this._popupEl.querySelector(".modal__button");
```

Comment:
`    this._submitBtn = this._popupEl.querySelector(".modal__button");
`

Please, delete the submit button from `Popup` because the image modal doesn't have it